styleguide/source/_patterns/02-molecules/inline-bio.twig

```diff
@@ -1,19 +1,21 @@
+{% set image, heading = bio.image, bio.heading %}
+
 <aside class="ama__inline-bio">
-  {% set image = bio.image %}
   {% if bio.image %}
-    {% include "@atoms/image.twig"  with {
-      "class": "ama__inline-bio__image"}
-    %}
+      {% include "@atoms/image.twig"  with {
+        "class": "ama__inline-bio__image"}
+      %}
   {% endif %}
 
-  {% set heading = bio.heading %}
-  {% include "@atoms/heading.twig" %}
+  <div class="ama__inline-bio__text">
```

Comment:
what was the purpose of this change? Seems irrelevant to this ticket.

Reply:
this was done because the article body copy needs align with the bio text above it per UX comp.